pytorch_cnn_trainer/engine.py

```diff
@@ -419,11 +467,15 @@ def train_sanity_fit(
                 print(f"Sanity check passed till {cnt} train batches")
                 print("All specicied batches done.")
                 train_sanity_end = time.time()
-                print(f"Training Sanity check passed in time {train_sanity_end - train_sanity_start} !!")
+                print(
+                    f"Training Sanity check passed in time {train_sanity_end - train_sanity_start} !!"
+                )
                 return True
 
     train_sanity_end = time.time()
-    print(f"Training Sanity check passed in time {train_sanity_end - train_sanity_start} !!")
+    print(
+        f"Training Sanity check passed in time {train_sanity_end - train_sanity_start} !!"
+    )
     return True
```

Comment:
Can the same logic be applied to `sanity_fit` too ?

Since users will expect the same API for both.

Reply:
yes I would implement it